cruise-control-client/cruisecontrolclient/util/print.py

```diff
@@ -0,0 +1,11 @@
+import sys
```

Comment:
Applies to all files -- can we append the following prefix to each file?
```
# Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License").
# See License in the project root for license information.
```

Reply:
Agreed; added